easybuild/easyconfigs/l/libxslt/libxslt-1.1.37-GCCcore-12.2.0.eb

```
@@ -24,10 +24,12 @@ dependencies = [
 ]
 
 # Make sure it doesn't pick up OS installed libgcrypt or Python
-configopts = '--with-crypto=no --with-python=no '
+# enable building static libs
+configopts = '--with-crypto=no --with-python=no --enable-static=yes '
 
 sanity_check_paths = {
-    'files': ['bin/xsltproc', 'include/libxslt/xslt.h', 'lib/%%(name)s.%s' % SHLIB_EXT],
+    'files': ['bin/xsltproc', 'include/libxslt/xslt.h', 'lib/%%(name)s.%s' % SHLIB_EXT, 'lib/%(name)s.a',
+              'lib/libexslt.%s' % SHLIB_EXT, 'lib/libexslt.a',],
     'dirs': [],
```

How about adding the include folders here:
```suggestion
    'dirs': ['include/libxslt', 'include/libexslt'],
```

---

added in 0890be1